streams/src/test/java/org/apache/kafka/test/MockProcessor.java

```diff
@@ -16,17 +16,17 @@
  */
 package org.apache.kafka.test;
 
-import java.time.Duration;
 import org.apache.kafka.streams.processor.AbstractProcessor;
 import org.apache.kafka.streams.processor.Cancellable;
 import org.apache.kafka.streams.processor.ProcessorContext;
 import org.apache.kafka.streams.processor.PunctuationType;
-import org.apache.kafka.streams.processor.Punctuator;
 
+import java.time.Duration;
 import java.util.ArrayList;
 
 import static org.junit.Assert.assertEquals;
 
+@SuppressWarnings("WeakerAccess")
```

Comment:
Just curious, why is this necessary?

Reply:
Intellij warns that some member (eg, `processedKeys`) are public.

Reply:
Sorry, my question was ambiguous. I'm wondering if, since this is a test artifact, we can consider it an internal API and therefore just make the method non-public.